applications/CoSimulationApplication/python_scripts/data_transfer_operators/copy_single_to_dist.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
```

Comment:
in general I would suggest to complete the name `copy_single_to_distributed` 
Also a description of the class and what it does would be very helpful for ppl who want to use it in future.